test/mongocontroller/controller.go

```diff
@@ -80,7 +97,9 @@ func New(p Params) (*Controller, error) {
 	if err != nil {
 		return nil, err
 	}
-	res := client.Database("foo").RunCommand(nil, map[string]int{"buildinfo": 1})
+
+	// test mongo connection
+	res := client.Database("foo").RunCommand(context.TODO(), map[string]int{"buildinfo": 1})
```

Comment:
Should this be context.Background()?

Reply:
I think context.Background() work as well.

Reply:
Hmm, here's how contexts are apparently used in the go driver:
https://www.mongodb.com/docs/drivers/go/current/fundamentals/context/

a TODO context implies that there should be some context there, but we don't know what it is. IMO in this situation we don't need any special context so Background makes more sense

Reply:
👍

Reply:
:+1: